apis/core/v1alpha1/resource.go

```diff
@@ -209,19 +209,20 @@ type ProviderSpec struct {
 	CredentialsSecretRef SecretKeySelector `json:"credentialsSecretRef"`
 }
 
-// A TargetSpec defines the common fields of objects that satisfy the Target interface.
+// A TargetSpec defines the common fields of objects used for exposing
+// infrastructure to workloads that can be scheduled to.
 type TargetSpec struct {
-	// SecretRef specifies the name of a Secret, in the same namespace as this
-	// target, to which any connection details for this target should be written
-	// or already exist. Connection secrets referenced by a target should
-	// contain information for connecting to a resource that allows for
-	// scheduling of workloads. +optional
-	SecretRef *LocalSecretReference `json:"secretRef,omitempty"`
-
-	// A ClusterRef specifies an existing managed resource, in any namespace, to
-	// which this target should attempt to propagate a connection secret from.
-	// +optional
-	ClusterRef *corev1.ObjectReference `json:"clusterRef,omitempty"`
+	// WriteConnectionSecretToReference specifies the name of a Secret, in the
+	// same namespace as this target, to which any connection details for this
+	// target should be written or already exist. Connection secrets referenced
+	// by a target should contain information for connecting to a resource that
+	// allows for scheduling of workloads. +optional
```

Comment:
Did the `+optional` stanzas get wrapped here (and on surrounding types)?

Reply:
Looks like it, will fix!